packages/calypso-build/webpack/file-loader.js

```diff
@@ -1,19 +1,26 @@
+const path = require( 'path' );
+
 /**
  * Return a Webpack loader configuration object for files / images.
  *
- * @param  {object} options File loader options
- * @returns {object}         Webpack loader object
+ * @param {object} options Rule.generator options
+ * @param options.outputPath Where to output the asset
+ * @param options.name Name of the asset
+ * @param options.publicPath Path used to generate the URL for the asset
+ * @param options.emitFile Opt out of writing assets
```

Comment:
Isn't this the other way around? Currently, if this is `true`, we're opting to write assets, not opting out of that.